### ISSUE TITLE
RileyLink connection failure

### ISSUE THREAD

**Issue by cauthren** [LoopKit/Loop#826]
How do you recommend uploading a report when the strings of text are so long? I've tried uploading as .txt file as well, but it is not working.

**Comment by Kdisimone** [comment]
Email the file (issue report) to yourself and then it is available as an attachmet where needed.

**Comment by Kdisimone** [comment]
I'm going to close this issue because there's no description of a problem or issue to troubleshoot beyond the original question.  If you have a follow-up issue, please post with more details about what you need help with and attach the log there.

**Comment by cauthren** [comment]
Hi Katie- have tried emailing to myself and attaching the file, but it
won’t attach (.txt file type not supported, also .md file types not
working). So I tried copying and pasting the text of the report in the body
of my post. I had to break it up into several different posts to fit.

The error measure is a “connection failure” (make sure RileyLink is nearby
and powered on). Have charged the Riley Link, but that’s not the issue.
It’s also turned on. Am running out of ideas. If you’re able to help at
all, that would be great.
